packages/embark/src/lib/modules/console/suggestions.ts

```diff
@@ -28,6 +29,36 @@ export default class Suggestions {
     this.listenToEvents();
   }
 
+  private get suggestions(): SuggestionsList {
+    if (this._suggestions.length) {
+      return this._suggestions;
+    }
+
+    this._suggestions.push({value: "web3.eth", command_type: "web3 object", description: "module for interacting with the Ethereum network"});
```

Comment:
Could the entries that are the same every time be stored in another property array like `__sugestions`? In other words, in the class definition it's a literal array instead of being built from a series of push statements. Then, when building `_suggestions` it would start with `this._suggestions = [...this.__suggestions]`.

Reply:
Agree, we can put that in a json file next to this one.

Reply:
@michaelsbradleyjr @jrainville - this should all be added in. Note, I decided to make the `DEFAULT_SUGGESTIONS` property `private static readonly` (and capitalize it for convention) as it will not be updated during runtime.